system/services/presideObjects/SqlSchemaSynchronizer.cfc

```diff
@@ -702,7 +702,29 @@ component {
 		return variables._tableCache[ arguments.dsn ][ arguments.tableName ] ?: false;
 	}
 
-	private query function _getTableColumns() {
+	private array function _getTableColumns() {
+		try {
+			if ( _getDbInfoService().hasModernDbInfo() ){
+				// with the faster version, we can fetch the entire schema in one call.
+				if ( !StructKeyExists( variables, "columnCache" )){
+					variables.columnCache =  _getTableColumnCache( argumentCollection = arguments );
```

Comment:
it's my understanding this only runs once at startup? but it probably makes sense to purge this cache after the sync finishes, no need for a query with all the columns in the db to be left hanging in memory